lexibank_tls.py

```diff
@@ -660,12 +660,7 @@ def cmd_makecldf(self, args):
             if entry["LGABBR"] in ["Note", "Gweno1"]:
                 continue
 
-            if entry["SRCID"].endswith(".0"):
-                src_idx = entry["SRCID"].strip(".0")
-            elif entry["SRCID"].endswith(".5"):
-                src_idx = entry["SRCID"].replace(".5", "a")
-            else:
-                src_idx = entry["SRCID"]
+            src_idx = entry["SRCID"].replace(".0", "").replace(".5", "a")
```

Comment:
Wait, sorry, but why do you replace 0.5 by "a"? Does this mean the entries with the number ending in 0.5 are resulting from numbers ending in -a?

Reply:
As far as I can tell, all of the `SRCID`s in the original data ending with `.5` correspond to the entries in the original list ending with `a`. I.e. `,269.5,mwika,Bungu,,kukwenda,,,,walk,,` in the original data is our `269a`.